README.md

```diff
@@ -5,6 +5,8 @@
 - deatch_addon.sh -> Deatch the ODF MS consumer addon from hive.
 - restore_consumer.sh -> Update the StorageConsumer id and StorageProviderEndpoint in the StorageCluster CR.
 - updateEBSVolumeTags.sh -> Update the aws ebs volume tags for osd's and mon's from provider cluster.
+- migrate.sh -> Will run all neccessory script to migrate/restore cluster.
```

Comment:
migrate script is different from restore, maybe we can have a separate script to restore the cluster in case of failure

Reply:
```suggestion
- migrate.sh -> Will run all necessary script to migrate cluster.
```